project-code/app/com/edulify/modules/geolocation/GeolocationFactory.java

```diff
@@ -0,0 +1,34 @@
+package com.edulify.modules.geolocation;
+
+/**
+ * Created by sowhat
+ */
+public class GeolocationFactory {
```

Comment:
But, since `Geolocation` is a simple POJO, do we really need a factory here? It looks like a overkill to me.